src/utl/README.md

```diff
@@ -0,0 +1,63 @@
+# Utilities
+
+The utility module houses of the manpage command in OpenROAD,
```

Comment:
I would reword this to "The utility module contains the man command.".  However it feels redundant given that the next thing is the man command itself.